back/engines/commercial/multi_tenancy/lib/tasks/core/clean_up_orphaned_image_records.rake

```diff
@@ -7,9 +7,18 @@ namespace :cl2back do
   # Execute (destroys records!): cl2back:clean_up_orphaned_image_records['execute']
   task :clean_up_orphaned_image_records, [:execute] => [:environment] do |_t, args|
     live_run = true if args[:execute] == 'execute'
+    tot_li = 0
```

Comment:
What do these crypto codes mean? :D

Reply:
They were longer and more descriptive, but because you wanted me to refactor to methods, I shortened them, so that the call to the methods would not be longer than the line it was replacing.

In other words, the methods don't do much to simplify the code.

Reply:
I've refactored again, and improved these var names again.